istioctl/cmd/kubeuninject.go

```diff
@@ -0,0 +1,373 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"reflect"
+
+	"github.com/ghodss/yaml"
+	openshiftv1 "github.com/openshift/api/apps/v1"
+	"github.com/spf13/cobra"
+	"go.uber.org/multierr"
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/api/batch/v2alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	yamlDecoder "k8s.io/apimachinery/pkg/util/yaml"
+
+	"istio.io/pkg/log"
+)
+
+const (
+	proxyContainerName              = "istio-proxy"
+	initContainerName               = "istio-init"
+	envoyVolumeName                 = "istio-envoy"
+	certVolumeName                  = "istio-certs"
+	annotationPolicy                = "sidecar.istio.io/inject"
+	annotationStatus                = "sidecar.istio.io/status"
+	annotationRewriteAppHTTPProbers = "sidecar.istio.io/rewriteAppHTTPProbers"
+)
+
+var (
```

Comment:
Please reuse the logic in pilot/pkg/kube/inject/initializer.go  We should have this list in two seperate places.
Also please reuse the codes in pilot/pkg/kube/inject/inject.go and do not write here again. eg: fromRawToObject(), etc.